ecmascript/parser/tests/jsx/basic/custom/unary-paren/input.js.json

```diff
@@ -10,7 +10,7 @@
       "type": "ImportDeclaration",
       "span": {
         "start": 0,
-        "end": 26,
+        "end": 27,
```

Comment:
Import declaration should include the last semi-colon. This test and some others seem to be failing on master.

Reply:
Currently travis ci chokes on other tests for parser.
(It seems like a rustc version issue)


So I'll merge the pr after testing locally.

Reply:
Oh I see. By the way, feel free to make any edits to fix any mistakes I made. I am going a bit blind here because II'm not sure how to run the tests for this. (I tried `cargo +nightly test --color always --all --all-features`, but it doesn't seem to run the input.json-like tests)

Reply:
cargo +nightly test --color always --all --all-features

invokes input.ts based reference testing, but it updates reference file when comparison failed. This behavior was introduced to avoid updating test one by one.